main.go

```diff
@@ -33,10 +33,10 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	metallbv1alpha1 "github.com/metallb/metallb-operator/api/v1alpha1"
-	metallbv1beta1 "github.com/metallb/metallb-operator/api/v1beta1"
 	"github.com/metallb/metallb-operator/controllers"
 	"github.com/metallb/metallb-operator/pkg/platform"
+	metallbv1alpha1 "go.universe.tf/metallb/api/v1alpha1"
```

Comment:
We'll have to remove the controllers too, as metallb will handle the CRs and the only bit the operator will handle is the deployment (i.e. the metallb resource and it's references)

Reply:
done.